pages/news/2020-XX-XX-new-in-2-3-rekordbox-support.html

```diff
@@ -0,0 +1,50 @@
+title: New in 2.3: Importing tracks and cues from Rekordbox removable devices
+author: ehendrikd
+date: 2020-05-15 20:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Do you rock your gigs armed only with your cans and a Rekordbox prepared USB flash drive? Ever been tempted to play your carefully curated tracks on something other than a CDJ or Rekordbox, on something that is free and open-source? We have news for you!
+
+Mixxx 2.3 now supports reading the following from Rekordbox prepared removable devices on all Mixxx’s supported platforms:
+
+* Folders
+* Playlists
+* Beatgrids
+* Hotcues
+* Memory cues
+* Loops
+
+This includes importing the key and BPM analyzed by Rekordbox, and comments and colors for tracks, cues and loops. **We have to be clear however:** this feature *only* reads Rekordbox prepared removable devices, such as USB flash drives or external hard drives. It *will not* read your locally stored Rekordbox collection if you also have Rekordbox installed. The main Rekordbox database (both for Rekordbox 5 and Rekordbox 6) use completely different formats from the USB drives, so reading those databases would be a whole other project.
+
+## Using the Rekordbox removable device library feature
+
+Assuming you already have a Rekordbox prepared removable device, simply plug it in while Mixxx is running. Click on Rekordbox in the library, and all found compatible removable devices will be visible (attach as many as you like!). Then click on the desired removable device, and all your folders and playlist that you have so tirelessly prepared will be revealed for you to play from.
+
+{# TODO: Insert screenshot of Rekordbox library feature once the skins are finalized. #}
+
+Some notes to keep in mind on how Rekordbox hotcues memory cues and loops are imported: Mixxx currently has one main cue point, one loop, and 36 hotcues. As such, all Rekordbox hotcues are mapped to Mixxx hotcues as expected, the first chronological Rekordbox memory cue is mapped to the Mixxx main cue, and the first chronological Rekordbox loop mapped to the Mixxx loop. All loops (including the first loop), and all subsequent Rekordbox memory cues are appended as Mixxx hotcues following the previously mapped actual hotcues. A little confusing? Perhaps, but hopefully all will make sense as you give it a whirl. Rest assured the hotcue and memory cue colors you assigned in Rekorbox are also imported, assisting in distinguishing between them. Also note that whilst the additional imported Rekordbox loops only appear in Mixxx 2.3 as hotcues, their loop details are preserved, ready for the multiple loop feature proposed for Mixxx 2.4!
```

Comment:
```suggestion
Some notes to keep in mind on how Rekordbox hotcues memory cues and loops are imported: Mixxx currently has one main cue point, one loop, and 36 hotcues. As such, all Rekordbox hotcues are mapped to Mixxx hotcues as expected, the first chronological Rekordbox memory cue is mapped to the Mixxx main cue, and the first chronological Rekordbox loop mapped to the Mixxx loop. All loops (including the first loop), and all subsequent Rekordbox memory cues are appended as Mixxx hotcues following the previously imported Rekordbox hotcues. A little confusing? Perhaps, but hopefully it will all make sense when you give it a whirl. Rest assured the hotcue and memory cue colors you assigned in Rekordbox are also imported, assisting in distinguishing between them. Also note that whilst the additional imported Rekordbox loops only appear in Mixxx 2.3 as hotcues, their loop details are preserved, ready for the multiple loop feature proposed for Mixxx 2.4!
```